sdk/core/azure-core-tracing-opencensus/azure/core/tracing/ext/opencensus_span/__init__.py

```diff
@@ -68,15 +68,15 @@ def span_instance(self):
         """
         return self._span_instance
 
-    def span(self, name="span"):
-        # type: (Optional[str]) -> OpenCensusSpan
+    def span(self, name="span", **kwargs):
```

Comment:
You are missing doccomment on what `kwargs` are used for...

Reply:
added